uSync.BackOffice/Services/SyncActionService.cs

```diff
@@ -86,13 +86,23 @@ public SyncActionResult ReportHandler(SyncActionOptions options, uSyncCallbacks
         return new SyncActionResult(actions);
     }
 
+    private string[] GetFolders(SyncActionOptions options)
```

Comment:
@KevinJump Also wasn't sure if you wanted to have validation on the `SyncActionOptions` to ensure that only one of `Folder` or `Folders` is defined?